lib/active_merchant/billing/gateways/trans_first_transaction_express.rb

```diff
@@ -156,7 +156,48 @@ class TransFirstTransactionExpressGateway < Gateway
       }
 
       EXTENDED_RESPONSE_MESSAGES = {
-        'B40K' => 'Declined Post – Credit linked to unextracted settle transaction'
+        'A400' =>	'Approved Post – Re-auth attempt approved and settled',
```

Comment:
Majority of these came from: https://github.com/rednovalabs/active_merchant/pull/9/commits/9d1e185f3013e272ed87c39bc509c53e80536994